### ISSUE TITLE
A11y_Visual Studio Code_Search_Focus order: Incorrect Keyboard Focus order is observed at Search and Replace controls 

### ISSUE THREAD

**Issue by v-kanve** [microsoft/vscode#122975]
GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code;#Focus Order;#Keyboard;#WCAG2.4.3;#Win32;#Win10;#AINotInScope;

### Environment Details:​
Application Name: Visual Studio Code
Standard Violated:WCAG2.4.3

### Repro Steps:​
1. Open Visual Studio Code editor
2. Tab till Search button in the view menu and press enter
3. Tab till search control
4. Now press tab again 
5. Verify whether keyboard focus is going to the next control Match or not

### Actual Result:
Incorrect Keyboard Focus order is observed at Search and Replace controls. After search control when we tab focus is moving to replace and again it is coming to match case control in the search control.

### Expected Result: ​
Keyboard focus should go to all the controls in the Search control. First it should go to search->Match case -> Match whole word ->Use regular expression. And then it should go to Replace controls. Focus order should be from left to right and from top to bottom.

### User Impact:
Keyboard only users will face difficulty while navigation if Keyboard focus is not correct.

### Attachment:
[A11y_Visual Studio Code_Search_Focus order Incorrect Keyboard Focus order is observed at Search and Replace controls.zip](https://github.com/microsoft/vscode/files/6425132/A11y_Visual.Studio.Code_Search_Focus.order.Incorrect.Keyboard.Focus.order.is.observed.at.Search.and.Replace.controls.zip)

**Comment by roblourens** [comment]
This is by design, we tab to replace first because it is the most commonly used control after the search input. All buttons and inputs are accessible by tabbing.

**Comment by InduPriya1805** [comment]
GitHubTags:#VisualStudioCode-Win32-Apr2021;
